### ISSUE TITLE
Stratified validation set

### ISSUE THREAD

**Issue by alvarouc** [fchollet/keras#796]
I am coming up with the problem that my validation set is getting all labels of one class, so the validation accuracy is either 0 or 1. 

Could someone point me where to change the validation split to use StratifedShuffleSplit from sklearn

http://scikit-learn.org/stable/modules/generated/sklearn.cross_validation.StratifiedShuffleSplit.html

**Comment by transcranial** [comment]
You can do a stratified split from sklearn, and then pass that in through `validation_data` in `.fit()` rather than `validation_split`

**Comment by alvarouc** [comment]
True!, thanks.

**Comment by eyaler** [comment]
would still be nice to add a flag for that

**Comment by alup** [comment]
:+1:
